process_manager/views/pages.py

```diff
@@ -32,10 +35,12 @@ def logs(request: HttpRequest, uuid: uuid.UUID) -> HttpResponse:
       The rendered page.
     """
     logs_response = get_process_logs(str(uuid))
-    context = dict(log_text="\n".join(val.data.line for val in logs_response))
-    return render(
-        request=request, context=context, template_name="process_manager/logs.html"
-    )
+
+    # Process the log text to exclude empty lines
+    log_lines = [val.data.line for val in logs_response if val.data.line.strip()]
+
+    context = {"log_lines": log_lines, "log_text": "\n".join(log_lines)}
```

Comment:
Seems `log_text` is no longer used in the template context and can be removed.

Reply:
`log_lines` gives more granular control for rendering each line, while `log_text` maintains compatibility with existing tests and code until it's fully transitioned to use `log_lines`.

Reply:
Ok, I'll remove it. Can you update the test?

Reply:
You can probably bundle it it's a simple change here to swap log_text for log_lines - https://github.com/ImperialCollegeLondon/drunc_ui/blob/99163d227022ad3d857ca15d87422795f8ca3412/tests/process_manager/views/test_page_views.py#L36

Reply:
Seems ok!